Dockerfile

```diff
@@ -0,0 +1,76 @@
+# A Dockerfile to build a development environment for
+#
+#         MM                       MMMMM
+#            MM                    MMMMMMM        MMM
+#                MMMM              MMMMMMMMM    MMMMM         MMMMMMMMMM
+#                    MM             MMMMMMMMM MMM  MM    MMMMMMMMMMMMMMM
+#        MMMMMM       M              MMMMMMMMMM    MM MMMMMMMMMMMMMMMMMM
+#              MM     M                MMMMMMMMM   MMMMMMMMMMMMMMMMMMMM
+#                MM  MMM        M       MMMMMMMMMMMMMMMMMMMMMMMMMMMMM
+#              MMMMMMMMMMMMM   MMMM       MMMMMMMMMMMMMMMMMMMMMMMMM
+#             MMMMMMMMMMMMMM  MMM MMMM   MMMMMMMMMMMMMMMMMMMMMMMM
+#             MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
+#           MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
+#          MMMMMMMMMMMMM   MMMMMMMMMMMMMMMM MMMMMMMMMMMMMMM   MMMMMMMM
+#          M  MMMM   MM    MMMMMMMM MMMMM     MMMMMMMMMMMMMMM       MM
+#            MMM    MM     MM   MMM   MMMM            MMMMMMMMM      MM
+#                  MM      M MMMMM       MMMMMM         MMMMMMMMMM    MM
+#                MMM    MMMMMM               MMMM       M MMMMMMMMMMMM MM
+#              MMM      MM M                  MM        MM   MMMMMMMMMMMMM
+#           MMM        MM  M                 MM         MM     MMMMMMMMMMMMMMM
+#         MM          MM   M               MM            MM       MMMMMMMMMMMMMM
+#       MMM         MM    MM            MMM               MMMM      MMMMMMMMMMMM
+#    MMM      MMM MMMMMMMMMMMMMMMMMMMM MMMMMMMMMMMMMMMMMMM  MM    MMM MMMMMMMMMM
+#    MMM      MMM MMMMMMMMMMMMMMMMMMMM MMMMMMMMM MMMMMMMMM  MMMM  MMM MMMMMMMMMM
+#    MMM          MMM    MMMMMM    MMM MMM   MMM   MMM MMM  MMMMMMMMM    MMMMMMM
+#    MMM      MMM MMMMMMMMM MMM    MMM MMMMMMMMM  MMM  MMM  MM MMMMMM    MMMM
+#    MMM      MMM MMMMMMMMMMMMMMMMMMMM MMMMMMMMM  MM   MMM  MM    MMM    MMMM
+#    MMM   MM MMM MMM    MMMMMM  MMM   MMM        MMMMMMMM  MM    MMM    MMMM
+#    MMMMMMMM MMM MMMMMMMMMMMMM   MMM  MMMMMMMMM  MM   MMM  MM    MMM    MMMM
+#    MMMMMMMM MMM MMMMMMMMMMMMM   MMM  MMMMMMMMM  MM   MMM  MM    MMM    MMMM
+#
+#
+# To build run the following from the root of the libreant directory:
+#
+#  docker build -t libreant-box .
+#
+# To start the container and operate inside it run:
+#
+#  docker run --privileged --rm -ti libreant-box
```

Comment:
Why `--priviledged`?

Reply:
Because we have to launch the services via ```service elasticsearch start``` and ```service libreantd start```.